lib/push.js

```diff
@@ -60,7 +60,7 @@ module.exports = function(file, options, callback) {
 	}
 
 	function getUri(options) {
-		var packageUri = url.resolve(options.host, '/api/packages/raw');
+		var packageUri = url.resolve(options.host, 'api/packages/raw');
```

Comment:
Moving to a relative path here as opposed to an absolute one fixes the issue.

I do not believe this would introduce any undesired behavior in the URI generation, but correct me if I am wrong on that.